src/panel-manager.js

```diff
@@ -7,8 +7,20 @@ genotet.panelManager = {};
 
 /** @const {number} */
 genotet.panelManager.COLLAPSED_WIDTH = 32;
-/** @const {number} */
-genotet.panelManager.TRANSITION_TIME = 300;
+/** @private @const {number} */
+genotet.panelManager.TRANSITION_TIME_ = 300;
+/** @private @const {number} */
+genotet.panelManager.TOGGLE_BTN_HEIGHT_ = 50;
+
+/**
+ * The difference between the width and other attributes of the tabs container.
```

Comment:
What is "the difference between the width and other attributes"? This is a very confusing sentence. I see no clue how to compare "width" and "other attribute", thus don't know what "difference" there is.